index.qmd

```diff
@@ -9,5 +9,5 @@ All submissions to the Image24 conference with additional figures, references, .
 List of abstrac:
 
 - [Digital twin with control](DigitalTwin/abstract.qmd) An uncertainty-aware digital twin for geological carbon storage
-
+- [Velocity Continuation for Common Image Gathers with Fourier Neural Operators](Rex2024SEG/paper.qmd) A framework to obtain CIGs in a fast manner
```

Comment:
A framework to accelerate migration-velocity analysis and uncertainty quantification